beta-src/src/components/ui/WDInfoDisplay.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from "react";
+import {
+  Box,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  TableContainer,
+} from "@mui/material";
+import GameMode from "../../enums/GameMode";
+import Season from "../../enums/Season";
+import Ranking from "../../enums/Ranking";
+import Phase from "../../enums/Phase";
+
+/**
+ * game setting datas which would be passed to the component by parent component/ context/redux store
+ */
+
+interface WDInfoDisplayProps {
+  gameMode: GameMode;
+  gameTime: number | React.ReactNode;
+  phase: Phase;
+  potNumber: number;
```

Comment:
`potNumber` should use the `IntegerRange` type. Since bet amount can only range from 5-95, this means that `potNumber` only have a possible range of 35 - 665.